src/document.h

```diff
@@ -2024,10 +2023,21 @@ struct Document {
         }
     }
 
+    const wxChar *LoadSearch() {

```

Comment:
Unless they're gigantic, I'd generally say there is no good reason introducing functions that only get called once, makes the code harder to follow

Reply:
Ok, I changed that.